Makefile

```diff
@@ -77,7 +77,8 @@ METADATA_VAR += CommitSHA=$(EXTRA_VERSION)
 METADATA_VAR += BaseDockerLabel=$(BASE_DOCKER_LABEL)
 METADATA_VAR += DockerNamespace=$(DOCKER_NS)
 
-GO_VER = 1.24.0
+GO_VER := $(shell grep '^go[ \t]' < go.mod)
```

Comment:
For people newer to Go and scripting, it may be nice to add a comment like:
`Go version maintained in the go.mod file`.

Reply:
Done